src/rabbitmq.js

```diff
@@ -57,19 +57,15 @@ class RabbitMQ {
     this.port = opts.port || parseInt(process.env.RABBITMQ_PORT, 10) || 5672
     this.username = opts.username || process.env.RABBITMQ_USERNAME
     this.password = opts.password || process.env.RABBITMQ_PASSWORD
-    this.log = logger.child({
-      module: 'ponos:rabbitmq',
-      hostname: this.hostname,
-      port: this.port,
-      clientName: this.name
-    })
+    this.log = opts.log || logger.child({ module: 'ponos:rabbitmq' })
```

Comment:
this is for the rabbitMQ module specifically (used as publisher) but ponos server uses the logger we passed in